opensearch-operator/opensearch-gateway/services/os_security_service.go

```diff
@@ -373,3 +374,88 @@ func DeleteActionGroup(ctx context.Context, service *OsClusterClient, actionGrou
 	}
 	return nil
 }
+
```

Comment:
Tenants is also a part security_service like the users, roles, rolesmapping etc. Since the rest are in this file, it also makes sense to put these here.